recipes/newrelic/cassandra/amazonLinux2.yml

```diff
@@ -0,0 +1,118 @@
+name: Cassandra Open Source Integration
```

Comment:
I think this file should be named "rhel.yml" since the platform it targets covers all the red hat linux platforms (amazon, centos, redhat).